tools/ci_build/build.py

```diff
@@ -2587,9 +2591,6 @@ def main():
     if args.use_tensorrt:
         args.use_cuda = True
 
-    if args.use_migraphx:
-        args.use_rocm = True
```

Comment:
MIGraphX CI fails since `NCCL` is not available when `use_rocm` is `False`.

Maybe add `OR onnxruntime_USE_MIGRAPHX` in line 1560?
https://github.com/microsoft/onnxruntime/blob/1b19045afadabd11c4534245eb547b20b72bb36e/cmake/CMakeLists.txt#L1558-L1562

Reply:
Sure committed. Lets see if that resolved things